cpp/include/torch_tensorrt/torch_tensorrt.h

```diff
@@ -15,6 +15,8 @@
 #include <string>
 #include <vector>
 
+#include <torch_tensorrt/macros.h>
```

Comment:
Just style wise, we use quotes for all the project internal includes

Reply:
This has been fixed in [this commit](https://github.com/pytorch/TensorRT/pull/1058/commits/c0bb7a8a18cfdfb84570891415c7fe66c604160b).